packages/react-vapor/src/components/table-hoc/TableRowConnected.tsx

```diff
@@ -198,10 +198,8 @@ class TableRowConnected extends React.PureComponent<
         _.chain(this.props.actions)
             .filter((action: IActionOptions) => action.callOnDoubleClick)
             .forEach((action: IActionOptions) => {
-                if (action.link) {
-                    window.location.href = action.link;
-                } else if (action.trigger) {
-                    action.trigger();
+                if (action.enabled) {
+                    action?.link ? (window.location.href = action.link) : action?.trigger();

```

Comment:
Personally I find it way easier to read with an if/else. Also you don't need the `?.` after action because it will always be defined. `trigger` might not though:

```suggestion
                    if (action.link) {
                        window.location.href = action.link
                    } else {
                        action.trigger?.()
                    }
```

Reply:
> Personally I find it way easier to read with an if/else. Also you don't need the `?.` after action because it will always be defined. `trigger` might not though:

The easier to read argument always wins in my book. I'll modify it accordingly